### PR TITLE
Use central feeds and fixup build

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

**Comment by dougbu** [comment]
@halter73 @wtgodbe @BrennanConroy I have no `push` rights in this repo. Please approve and, when the build is successful, merge. Thanks❕

**Review by wtgodbe — APPROVED** [review]

File: src/Microsoft.AspNet.SignalR.JS/.npmrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin
+registry=https://pkgs.dev.azure.com/dnceng/internal/_packaging/internal_PublicPackages/npm/registry/
```
Comment (BrennanConroy) [inline src/Microsoft.AspNet.SignalR.JS/.npmrc:2]: ?

Comment (dougbu) [reply src/Microsoft.AspNet.SignalR.JS/.npmrc:2]: It's a "special" feed that only authorized users can add packages to. Intended to avoid some issues.

Comment (dougbu) [reply src/Microsoft.AspNet.SignalR.JS/.npmrc:2]: And, yes, I added everything needed here to the feed

File: Directory.Build.props [hunk 2]
```diff
@@ -50,14 +49,12 @@
     <!-- Assembly/NuSpec attributes -->
     <Company>Microsoft</Company>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/SignalR/SignalR/2.3.0/LICENSE.txt</PackageLicenseUrl>
```
Comment (dougbu) [inline Directory.Build.props:53]: This pointed to the same content as in the local LICENSE.txt file

**Review by dougbu — COMMENTED** [review]
/ping reviewers

**Comment by BrennanConroy** [comment]
Getting auth failures on public feeds? Any idea what's going on?

**Comment by dougbu** [comment]
I haven't figured this one out yet @BrennanConroy. I suspect either I missed a directory when ingesting packages or `npm install --include=optional` didn't actually install everything.

**Comment by dougbu** [comment]
I suspect something changed since the last package-lock.json update. Changed the `install` command in the test folder to `ci` because that works a lot like `yarn install --frozen-lockfile` (though w/ less caching).

**Comment by dougbu** [comment]
Build is green. Plz do the (squash and) merge thing for me

**Pull request merged**